addons/web/static/src/views/fields/field.js

```diff
@@ -161,7 +161,7 @@ export class Field extends Component {
 }
 Field.template = "web.Field";
 
-Field.parseFieldNode = function (node, models, modelName, viewType, jsClass) {
+Field.parseFieldNode = function (node, models, modelName, viewType, jsClass, isEditable = true) {
```

Comment:
typically, isEditable rather refer to a view being editable (like list or form). When talking about access rights, we often use canEdit